telematic_system/telematic_historical_data_processing/rosbag2_processing_service/rosbag2_processing_service/rosbag_processor.py

```diff
@@ -89,23 +90,27 @@ def process_rosbag(self,rosbag2_path):
                     self.write_api.write(bucket=self.config.influx_bucket, org=self.config.influx_org, record=record)
 
                 except InfluxDBClientError as e:
-                    self.logger.error(f"Error from Influx Client: {(e)}")
+                    self.config.logger.error(f"Error from Influx Client: {(e)}")
                 except Exception as e:
-                    self.logger.error(f"Failed to process ros message with exception: {(e)}")
+                    self.config.logger.error(f"Failed to process ros message with exception: {(e)}")
 
 
         except exceptions.McapError as e:
-            self.logger.error(f"Failed to read from rosbag with exception {(e)} ")
+            processing_error_msg = f"Failed to read from rosbag with exception {(e)} "
+            self.config.logger.error(processing_error_msg)
             self.is_processing = False
-            return
 
-        self.logger.info(f"Completed rosbag processing for {rosbag2_name}")
+            return ProcessingStatus.ERROR.value, processing_error_msg
+
+        self.config.logger.info(f"Completed rosbag processing for {rosbag2_name}")
 
         self.is_processing = False
 
+        return ProcessingStatus.COMPLETED.value, "NA"
+
     def create_record_from_msg(self, msg, measurement_name):
 
-        topic = msg.channel.topic
+        topic = msg.channel.topic[1:]
```

Comment:
Why do we remove the first character?

Reply:
I removed it because the first character is "/", but no strong opinions on it. Looks like the vehicle bridge doesn't remove it, so we can keep it here as well